docs/src/plugin_deployment/plugin_deployment_using_chart_in_bundle.md

```
@@ -21,7 +21,7 @@ helm install -n lb-csi-ns --create-namespace lb-csi helm/lb-csi/
 helm list --namespace=kube-system
 
 NAME  	NAMESPACE  	REVISION  UPDATED        	STATUS  	CHART              	 APP VERSION
-lb-csi	kube-system	1         2025-05-21... 	deployed	lb-csi-plugin-1.19.0	 1.18.0
+lb-csi	kube-system	1         2025-07-16... 	deployed	lb-csi-plugin-v1.20.0	 1.18.0
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 7

(MD010, no-hard-tabs)